pkg/image-manager/handlers.go

```diff
@@ -539,15 +539,16 @@ func (h *Handlers) deleteImageByName(params image.DeleteImageByNameParams, princ
 				Message: swag.String("image not found"),
 			})
 	}
-	err = h.Store.Delete(ctx, params.XDispatchOrg, params.ImageName, &Image{})
+	e.Delete = true
+	_, err = h.Store.Update(ctx, e.Revision, &e)
```

Comment:
Does the controller come around and actually delete the image?

Reply:
From what I checked it does. I tried logging out all the entities from the store and after calling update, it gets removed from the store. It looks like all our other resources get deleted by calling `update` instead of `delete`. When calling `delete` it throws an error.